dash/development/component_generator.py

```diff
@@ -22,14 +22,17 @@ class _CombinedFormatter(argparse.ArgumentDefaultsHelpFormatter,
 
 # pylint: disable=too-many-locals
 def generate_components(components_source, project_shortname,
-                        package_info_filename='package.json'):
+                        package_info_filename='package.json',
+                        ignore='^_'):
```

Comment:
This covers the `__tests__` case mentioned in https://github.com/plotly/dash/issues/486 but maybe a small set of defaults (e.g `['__tests__, 'tests']) to cover both the dash-bootstrap-components and our own dash-component-boilerplate case would make sense.

Reply:
This is an additional feature and should be documented in the changelog and the version be bumped.